clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractMembershipManager.java

```diff
@@ -1012,12 +801,7 @@ void maybeReconcile() {
         // be retried until it succeeds, fails with non-retriable error, or timer expires.
         CompletableFuture<Void> commitResult;
 
-        // Issue a commit request that will be retried until it succeeds, fails with a
-        // non-retriable error, or the time limit expires. Retry on stale member epoch error, in a
-        // best effort to commit the offsets in the case where the epoch might have changed while
-        // the current reconciliation is in process. Note this is using the rebalance timeout as
-        // it is the limit enforced by the broker to complete the reconciliation process.
-        commitResult = commitRequestManager.maybeAutoCommitSyncBeforeRevocation(getDeadlineMsForTimeout(rebalanceTimeoutMs));
+        commitResult = signalReconciliationInProgress();
```

Comment:
I will change `signalReconciliationInProgress` to `signalReconciliationStarted`, and `signalReconciliationCompleted` to `signalReconciliationCompleting`. I am avoiding "on" because that clashes with the names of the user callbacks.